cleanlab_studio/studio/trustworthy_language_model.py

```diff
@@ -47,8 +47,6 @@ class TLM:
     After you've instantiated the TLM object using [`Studio.TLM()`](../studio/#method-tlm), you can use the instance methods documented on this page. Possible arguments for `Studio.TLM()` are documented below.
 
     Args:
-        api_key: You can find your API key on your [account page](https://app.cleanlab.ai/account) in Cleanlab Studio. Instead of specifying the API key here, you can also log in with `cleanlab login` on the command-line.
```

Comment:
The TLM class itself does require `api_key` though. But agree to put it here might be confusing since the studio.TLM() function doesn't require and we hide the __init__ of this class.